Octokit.Reactive/Clients/IObservablePullRequestReviewClient.cs

```diff
@@ -0,0 +1,190 @@
+        IObservable<PullRequestReview> Submit(string owner, string name, int pullRequestId, int reviewId, PullRequestReviewSubmit submitMessage);
+
+        /// <summary>
+        /// Submits an event to a pull request review.
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/pulls/reviews/#submit-a-pull-request-review</remarks>
+        /// <param name="repositoryId">The Id of the repository</param>
+        /// <param name="pullRequestId">The pull request review comment number</param>
+        /// <param name="reviewId">The pull request review number</param>
+        /// <param name="submitMessage">The message and event being submitted for the review</param>
+        IObservable<PullRequestReview> Submit(long repositoryId, int pullRequestId, int reviewId, PullRequestReviewSubmit submitMessage);
+
+        /// <summary>
+        /// Dismisses a pull request review.
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/pulls/reviews/#dismiss-a-pull-request-review</remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="pullRequestId">The pull request review comment number</param>
+        /// <param name="reviewId">The pull request review number</param>
+        /// <param name="dismissMessage">The message indicating why the review was dismissed</param>
+        IObservable<PullRequestReview> Dismiss(string owner, string name, int pullRequestId, int reviewId, PullRequestReviewDismiss dismissMessage);
+
+        /// <summary>
+        /// Dismisses a pull request review.
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/pulls/reviews/#dismiss-a-pull-request-review</remarks>
+        /// <param name="repositoryId">The Id of the repository</param>
+        /// <param name="pullRequestId">The pull request review comment number</param>
+        /// <param name="reviewId">The pull request review number</param>
+        /// <param name="dismissMessage">The message indicating why the review was dismissed</param>
+        IObservable<PullRequestReview> Dismiss(long repositoryId, int pullRequestId, int reviewId, PullRequestReviewDismiss dismissMessage);
+
+        /// <summary>
+        /// Lists comments for a single review
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/pulls/reviews/#get-comments-for-a-single-review</remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="pullRequestId">The pull request review comment number</param>
+        /// <param name="reviewId">The pull request review number</param>
+        IObservable<PullRequestReviewComment> GetAllComments(string owner, string name, int pullRequestId, int reviewId);
```

Comment:
ok I was mostly going off the API docs which showed pagination support in the response
<img width="860" alt="screen shot 2017-08-07 at 7 44 22 am" src="https://user-images.githubusercontent.com/13208452/29031834-5938a854-7b44-11e7-9196-6d1997d73626.png">